package org.SyncTask.exceptions;

public class UserNotFoundException extends Exception {
    public UserNotFoundException(String s) {
        super("Nenhum usu√°rio criado no banco de dados");
    }
}
